hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -131,11 +130,9 @@ public int getVersion() {
 
 	@Override
 	public int getMinimumChildCount(int version) {
-		if (version < RELEASE_0160_VERSION) {
-			return StateChildIndices.NUM_PRE_0160_CHILDREN;
-		} else if (version <= StateVersions.RELEASE_0180_VERSION) {
+		if (version < RELEASE_TWENTY_VERSION) {
 			return StateChildIndices.NUM_PRE_TWENTY_CHILDREN;
-		} else if (version <= RELEASE_TWENTY_VERSION) {
+		} else if (version == RELEASE_TWENTY_VERSION) {
```

Comment:
not directly related to the changes in this PR, but I think for consistency, we should rename the field to `RELEASE_0200_VERSION`